Src/WitsmlExplorer.Frontend/components/Modals/LogCurvePriorityModal.tsx

```diff
@@ -107,7 +131,6 @@ export const LogCurvePriorityModal = (
                 value={newCurve}
```

Comment:
Can you change what happens if you hit _Enter_ within the textfield? I want it to add the new curve and stop the propagation that closes the modal.

Reply:
done